tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+# that the Raw object of this sample dataset is lowpass filtered at 40Hz.
+
+# First we will xxtract the data from right trials in contralateral hemisphere
+
+# Define channel to use
+chan = 'EEG 054'
+
+# Copy and filter evoked
+right_vis_evoked = evokeds['visual/right'].copy()
+right_vis_evoked.filter(None, 20)
+right_vis_evoked.pick(chan)
+
+# Get peak latency and amplitude at EEG 054
+_, lh_lat, lh_amp = right_vis_evoked.get_peak(tmin=.07, tmax=.13,
+                                              mode='pos',
+                                              return_amplitude=True)
+
+# Convert amplitude from volts to microvolts
+lh_microvolts = lh_amp * 1e6
+
+# Plot the evoked trace with a point for the peak
+fig, ax = plt.subplots(nrows=1, ncols=1)
+fig = right_vis_evoked.plot(picks=chan, axes=ax,
+                            titles=f'Visual/Right Peak at {chan}',
+                            proj=True)
+ax.plot(lh_lat, lh_microvolts, 'ro')
+fig
+
+# Next, we will do the same for the left visual trials in
+# the right hemisphere
+chan = 'EEG 059'
+left_vis_evoked = evokeds['visual/right'].copy()
+left_vis_evoked.filter(None, 20)
+left_vis_evoked.pick(chan)
+_, rh_latency, rh_amplitude = left_vis_evoked.get_peak(tmin=.07, tmax=.13,
+                                                       mode='pos',
+                                                       return_amplitude=True)
+
+# Convert amplitude from volts to microvolts
+rh_microvolts = rh_amplitude * 1e6
+
+# Plot the evoked trace with a point for the peak
+fig, ax = plt.subplots(nrows=1, ncols=1)
+fig = right_vis_evoked.plot(picks=chan, axes=ax,
+                            titles=f'Visual/Left Peak at {chan}',
+                            proj=True)
+ax.plot(rh_latency, rh_microvolts, 'ro')
+fig
+
+
+###############################################################################
+# Obtaining Mean Amplitude in a Specified Time Window
+# ---------------------------------------------------------------
+#
+# Another common practice in ERP studies is to define a component (or effect)
+# as the mean amplitude within a specified time window. Sometimes, this also
+# involves extracting the mean over both time and space (sensors). The
+# following demonstrates how to do both. We will focus again on visual trials
+# in posterior electrodes.
+#
+# Below, we extract the mean amplitude from .08s to .12s for visual trials.
+# Given how the average amplitude over a time window is extracted, there is no
+# need to apply any additional lowpass filtering to the evoked data.
+
+# Define electrode clusters
+lh_cluster = [f'EEG {x:03}' for x in [44, 45, 54, 57]]
+rh_cluster = [f'EEG {x:03}' for x in [52, 55, 56, 59]]
+
+# Extract mean amplitude and store in a dictionary
+tmin, tmax = .08, .12
+
+# Create dictionary with amplitudes converted to microvolts
+mean_amplitude = {}
+for cond in ['visual/left', 'visual/right']:
+    lh_evoked = evokeds[cond].copy().pick(lh_cluster)
```

Comment:
`evokeds[cond]` should already give you a copy so `.copy()` seems unnecessary.

Reply:
I'm not so sure. Anytime I've check it without the copy it modifies the evoked list item in-place. Nonetheless, I dropped this bit of code in the most recent commit. There are other places I use the `.copy` method, so please feel free to point out if where I put them is not needed (highly likely I missed something).